tests/WebAssertTest.php

```diff
@@ -982,7 +982,7 @@ public function testElementAttributeExists()
         );
     }
 
-    public function testElementAttributeNotContains()
+    public function testElementAttributeNotExists()
```

Comment:
Are you certain, that you've added new tests and not changed existing ones? The diff shows me, that you've renamed existing tests (ones, existed before this PR).